moveit_ros/moveit_servo/include/moveit_servo/collision_check.h

```diff
@@ -86,19 +87,20 @@ class CollisionCheck
   planning_scene_monitor::LockedPlanningSceneRO getLockedPlanningSceneRO() const;
 
   /** \brief Callback for stopping time, from the thread that is aware of velocity and acceleration */
```

Comment:
Can you add the word "collision" here? This can be kind of confusing.

`/** \brief Callback for collision stopping time, from the thread that is aware of velocity and acceleration */`